repository/git.go

```diff
@@ -809,179 +1056,133 @@ func (repo *GitRepo) PullNotes(remote, notesRefPattern string) error {
 	return repo.MergeNotes(remote, notesRefPattern)
 }
 
-func getRemoteArchiveRef(remote, archiveRefPattern string) string {
-	relativeArchiveRef := strings.TrimPrefix(archiveRefPattern, "refs/devtools/archives/")
-	return "refs/devtools/remoteArchives/" + remote + "/" + relativeArchiveRef
+func getRemoteDevtoolsRef(remote, devtoolsRefPattern string) string {
+	relativeRef := strings.TrimPrefix(devtoolsRefPattern, "refs/devtools/")
+	return "refs/remoteDevtools/" + remote + "/" + relativeRef
```

Comment:
It's not super obvious to me why this wouldn't use a similar ref pattern to notes... like `refs/devtools/remotes/$remote/$relativeRef`

Reply:
Added comments explaining it. It's actually the notes refs that are the exception. The "refs/devtools/"->"refs/remoteDevtools/" naming follows the same pattern of "refs/heads/"->"refs/remotes/".